docs/mobile-apps/automated-testing/espresso-xcuitest/espresso-capture.md

```diff
@@ -0,0 +1,49 @@
+---
+id: espresso-capture
+title: Espresso Screenshot Capture
+sidebar_label: Espresso Screenshot Capture
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Espresso Screenshot Capture functionality allows you to easily take a snapshot of the error as a part of debugging process. These screenshots also help in in identifying any errors or bugs within your application.
+
+## What You'll Need
+
+* A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up))
+* A native Android, iOS, or iPadOS mobile app
```

Comment:
do we need to add this in every page? It's only for Android so please remove iOS and iPadOS